rm-datasource/src/main/java/com/alibaba/fescar/rm/datasource/exec/AbstractDMLBaseExecutor.java

```diff
@@ -64,7 +69,12 @@ protected T executeAutoCommitTrue(Object[] args) throws Throwable {
                 }
             }
 
-        } finally {
+        } catch(Exception e) {
```

Comment:
what exception will be caught here?

Reply:
some runtimeException besides LockConflictException, just like timeout in connectionProxy.commit(), other failure in executeAutoCommitFalse(args)。

I add catch statement here because  some exception occur in finally statement connectionProxy.setAutoCommit(true) DID cover up the orignal exception